src/shared/components/header/header.css

```diff
@@ -0,0 +1,162 @@
+.nav-container{
+  display: flex;
+  min-height: 12vh;
+  padding:0.8rem 1.2rem 0.8rem 0.8rem;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 2px 2px 5px rgb(0 0 0 / 20%);
+  background-color:var(--light-color);
+  position: sticky;
+  top:0;
+  z-index:10;
+  width:100%;
+}
+.logo {
+  margin: 0 1rem;
+}
+.left-navbar {
+  display: flex;
+}
+.navbar-link-list-item {
+  margin: 0 1rem;
+}
+.right-navbar {
+  display: flex;
+  align-items:center;
+}
+.navbar-links {
+  display: flex;
+  align-items: center;
+}
+.badge-icon{
+  font-size: 1.3rem ;
+}
+.search-bar-container {
+  display: flex;
+  align-items: center;
+  margin:0 0.5rem;
+  background-color: rgb(221, 221, 221);
+  border-radius: 0.3rem;
+}
+.search-bar-container button {
+  height: 2rem;
+  padding: 0 0.5rem;
+  margin: 0;
+  border: none;
+  background-color: transparent;
+}
+.search-bar {
+  height: 2rem;
+  background-color: transparent;
+  border: none;
```

Comment:
background-color: transparent is used multiple times, you could make a common class for that

Reply:
As we have some CSS for button and search-bar then by unnecessary making one another common class just for the sake of one property i.e background-color : transparent